### PR TITLE
feat: add new doc related page view events

### PR DESCRIPTION
**Pull request by yuhao900914**
# Amplitude Developer Docs PR

## Description
1. add the new clickIds in docs. https://amplitude.atlassian.net/wiki/spaces/GOV/pages/2058289286/Browser+SDK#Cookie-Usage
2. add details for page view events

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sdks/marketing-analytics-browser/index.md [hunk 1]
```diff
@@ -90,6 +90,18 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 
 --8<-- "includes/sdk-ts-browser/marketing-analytics.md"
 
+The following are the event info got tracked in the page view events
```
Comment (justin-fiedler) [inline docs/data/sdks/marketing-analytics-browser/index.md:93]:
```suggestion
The following information is tracked in the page view events.
```

File: docs/data/sdks/typescript-browser/index.md [hunk 2]
```diff
@@ -163,6 +164,18 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 });
 ```
 
+The following are the event info got tracked in the page view events.
```
Comment (justin-fiedler) [inline docs/data/sdks/typescript-browser/index.md:167]: Maybe should move this to an include to make it easier to keep in sync in all locations.

**Review by justin-fiedler — APPROVED** [review]
Overall looks good. See nits.

File: docs/data/sdks/typescript-browser/index.md [hunk 3]
```diff
@@ -163,6 +164,18 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
...
+
+|<div class="big-column">Name</div>| Description| Default Value|
+|---|----|---|
+|`event_type`| `string`. The event type for page view event. Configurable through `defaultTracking.pageViews.eventType` or enrichment plugin. | `[Amplitude] Page Viewed`, `[Amplitude] Page View` for version previous 1.9.1 |
```
Comment (kevinpagtakhan) [inline docs/data/sdks/typescript-browser/index.md:171]: > [Amplitude] Page View` for version previous 1.9.1

We don't have page view tracking under default tracking, so I think better to remove this

Comment (yuhao900914) [reply docs/data/sdks/typescript-browser/index.md:171]: https://github.com/amplitude/Amplitude-TypeScript/blob/5590f7d77d93362d2f3db6d47116ca502afd1fc4/packages/analytics-browser/src/browser-client.ts#LL121C41-L121C64

v1.9.0 do have defaultTracking.

Comment (yuhao900914) [reply docs/data/sdks/typescript-browser/index.md:171]: - v1.8.0 is using 'Page View' as the event type. Check [here](https://github.com/amplitude/Amplitude-TypeScript/blob/e9b73ad53882d499ca14e3d53e8885825554d0d9/packages/analytics-browser/src/browser-client.ts#L87) and [here](https://github.com/amplitude/Amplitude-TypeScript/blob/e9b73ad53882d499ca14e3d53e8885825554d0d9/packages/analytics-client-common/src/attribution/campaign-tracker.ts#L84-L91) if not using pageview plugin. 
- If we use the web attribution plugins, the page view event will be [this](https://github.com/amplitude/Amplitude-TypeScript/blob/e9b73ad53882d499ca14e3d53e8885825554d0d9/packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts#L47),  and [this is the event properties.](https://github.com/amplitude/Amplitude-TypeScript/blob/e9b73ad53882d499ca14e3d53e8885825554d0d9/packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts#L109-L118) 
- for the version between v1.9.0 - 1.9. the page view event type will be  `[Amplitude] Page View`. Check [here](https://github.com/amplitude/Amplitude-TypeScript/blob/5590f7d77d93362d2f3db6d47116ca502afd1fc4/packages/analytics-browser/src/browser-client.ts#LL121C41-L121C64). 
Due to the complicity of the event page view in browser SDK. Should we always only mention the latest version in this doc? 
wdyt? @justin-fiedler @Mercy811

Comment (kevinpagtakhan) [reply docs/data/sdks/typescript-browser/index.md:171]: v1.9.0 is considered a bad release so I'll lean on not putting this in a doc since we won't recommend it anyway. The default events tracked there are not planned in Data too.

Comment (kevinpagtakhan) [reply docs/data/sdks/typescript-browser/index.md:171]: For v1.8.x and below, page views are tracked under `attribution` and not under `defaultTracking`. My concern was that this section was for configuration under `defaultTracking` and so mentioning here seems misleading

File: docs/data/sdks/typescript-browser/index.md [hunk 4]
```diff
@@ -163,6 +164,18 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
...
+|`event_properties.page_domain`| `string`. The page domain. | location.hostname or ''. |
```
[the remark this answers is not in the document]
Comment (yuhao900914) [reply docs/data/sdks/typescript-browser/index.md]: Screen Shot 2023-05-17 at 3 07 46 PM
Update to this.

File: docs/data/sdks/typescript-browser/index.md (hunk 3 above)
Comment (yuhao900914) [reply docs/data/sdks/typescript-browser/index.md]: I rephrase this to `[Amplitude] Page Viewed` from version 1.9.1.

**Comment by yuhao900914** [comment]
@justin-fiedler  Asking you to re-reviewing this pr since there have some changes afterward.

File: docs/data/sdks/typescript-browser/index.md (hunk 3 above)
Comment (justin-fiedler) [reply docs/data/sdks/typescript-browser/index.md]: I think we should focus on the latest functionality. What about adding annotation to `Page Views` that explains the feature used to be on `attribution` and had different names for the events?

image

**Review by Mercy811 — APPROVED** [review]

**Review by justin-fiedler — APPROVED** [review]

**Review by bgiori — APPROVED** [review]
